index.html

```diff
@@ -0,0 +1,286 @@
+                           <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/peaceIcon.svg" alt=""></a>
+                       </li>
+                   </ul>
+               </div>
+               <br>
+               <button type="button"><img src="images/Getmyresume.png" alt=""></button>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Languages.png" alt=""></li>
+                   <li class="greaterthan"><a href="">&#86;</a></li>
+                </ul>
+               </div>
+               <div class="frameworks">
+                   <ul>
+                       <li><img src="images/Frame9.png" alt=""></li>
+                       <li><img src="images/Frame10.png" alt=""></li>
+                       <li><img src="images/Frame11.png" alt=""></li>
+                   </ul>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Frameworks1.png" alt=""></li>
+                   <li class="lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Skills1.png" alt=""></li>
+                   <li class="skills-text lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+           </div>       
+           </section>
+           <section class="works-2">
+            <div class="headline1">
+              
+                 <h2 class="mainheader">
+                     Contact Me</h2>
+                 <p class="details-2">
+                   <img src="images/PrimaryText3.png" alt="">
+                 </p>
+               
+               <div class="form-input">
+                <form action="https://formspree.io/f/mayvkezq" method="post">
+                    <div class="container">
+                      <input type="text" placeholder="Name" name="Name" id="name" maxlength="30">                
+
+                      <input type="email" placeholder="Email" name="email" id="email" required>
+                  
+                      <textarea class="textbox" placeholder="Comments" name="comments" id="comments" maxlength="500"></textarea>                                      
+                      
+                    </div>           
+                    <button type="submit" class="registerbtn">Get it Touch</button>    
+                    
+                  </form>
+               </div>   
```

Comment:
- [ ] The previous reviewer mentioned that the form design is not the same as the Figma design, kindly modify it so it's closer to the provided design as possible

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_072236](https://user-images.githubusercontent.com/25227022/172779727-9a325b82-827c-469c-a11a-a0f4da89e9ec.png)|<img width="310" alt="172643807-8d4f0a88-5653-4d20-923d-82318bbf6df2" src="https://user-images.githubusercontent.com/25227022/172779815-6022d630-1096-47bb-9990-cb5e3f13d575.png">|